intermediate-goals.Rmd

```diff
@@ -0,0 +1,148 @@
+# Intermediate Outline {#intermediate-outline}
+
+This outline describes the questions that the intermediate course will answer.
+It assumes learners have completed the novice course, 
+which means they know:
+
+-   how to write code (in Python or R) that reads, analyses and plots a dataset,
+-   how to write and run unit tests,
```

Comment:
As per #75, they will not know unit testing. (I predict the proposal will get passed :wink: )